src/apps/Scripts/BaseScripts.js

```diff
@@ -66,6 +66,22 @@ const baseScripts = {
       # )
 
       # print (new_user)`
+  },
+  base_php: {
+    source: `
+      // Start coding!
+      // Note: You do not need opening / closing PHP tags in this script
+
+      $count = count($CONFIG);
+
+      foreach ($CONFIG as $key => $val) {
+        if (--$count < 0) {
+          break;
+        }
+
+        echo "$key = $val\n";
```

Comment:
to get a new line after each value. otherwise we get sth like key1: val1 key2:val2 key3:val3 instead of 
key1:val1
key2:val2
key3:val3  

ofc I can adjust it if you have any suggestions.